sycl/CMakeLists.txt

```diff
@@ -104,6 +104,7 @@ if( NOT OpenCL_LIBRARIES )
                ${OPENCL_ICD_LOADER_WDK}
     STEP_TARGETS      configure,build,install
     DEPENDS           ocl-headers
+    BUILD_BYPRODUCTS ${LLVM_BINARY_DIR}/lib/libOpenCL.so
```

Comment:
Do we care about Windows?

Reply:
Yes.

Reply:
You won't have libOpenCL.so as a build product on Windows. Shouldn't it be OpenCL.lib or OpenCL.dll?

Reply:
Fixed.